eisbuk-admin/src/components/atoms/IntervalCard/IntervalCard.tsx

```diff
@@ -149,10 +124,10 @@ const BookingCard: React.FC<Props> = ({
               <Button
                 className={classes.actionButton}
                 onClick={handleClick}
-                color="secondary"
+                color={booked ? "secondary" : "primary"}
                 variant="contained"
               >
-                Cancel
+                {interval.startTime} - {interval.endTime}{" "}
```

Comment:
This should be `BookingCard.CancelBooking` or `BookingCard.BookInterval`. But, for typos/testing consistancy, save it in a `@/lib/labels` as `const __cancelBooking__ = "BookingCard.CancelBooking"`, etc.

Reply:
The time interval was ok as where it was. Check the storybook, the styles have been set up and shouldn't be changed (at least not for this PR).

Reply:
Also the button color was fine as it was. For now.